Topic-Array_methods/example.js

```diff
@@ -0,0 +1,202 @@
+
+const fruits3 = ["Banana", "Orange", "Apple", "Mango"];
+let s2 = fruits3.join(" * ");
+console.log(s2);
+
+/*..............................................................................................................*/
+
+const fruits4 = ["Banana", "Orange", "Apple", "Mango"];
+fruits4.pop();
+
+/*........................................................................................................*/
+
+const fruits5 = ["Banana", "Orange", "Apple", "Mango"];
+fruits5.push("Kiwi");
+
+/*...........................................................................................................*/
+
+const fruits6 = ["Banana", "Orange", "Apple", "Mango"];
+fruits6.shift();
+
+/*.................................................................................................*/
+const fruits7 = ["Banana", "Orange", "Apple", "Mango"];
+fruits7.unshift("Lemon");
+
+/*............................................................................................................*/
+
+const myGirls = ["Cecilie", "Lone"];
+const myBoys = ["Emil", "Tobias", "Linus"];
+
+const myChildren = myGirls.concat(myBoys);
+
+
+/*..........................................................................................................................*/
+
+
+const fruits8 = ["Banana", "Orange", "Apple", "Mango"];
+fruits8.copyWithin(2, 0); 
+
+/*...............................................................................................................*/
+
+const myArr = [[1,2],[3,4],[5,6]];
+const newArr = myArr.flat();
+console.log(newArr);
+
+/*..................................................................................................................*/
+
+
+const myArr1 = [1, 2, 3, 4, 5, 6];
+const newArr1 = myArr1.flatMap(x => [x, x * 10]);
+
+
+/*.................................................................................................................*/
+
+
+
+const fruits9 = ["Banana", "Orange", "Apple", "Mango"];
+console.log(fruits9.splice(2, 0, "Lemon", "Kiwi"));
+
+/*.................................................................................................................*/
+
+
+const fruits10 = ["Banana", "Orange", "Lemon", "Apple", "Mango"];
+const citrus = fruits10.slice(1, 3);
+console.log(citrus);
+
+
+/*...................................................................................................................*/
+
+const fruits11 = ["Apple", "Orange", "Apple", "Mango"];
+let position = fruits11.indexOf("Apple");
+
+/*.....................................................................................................................*/
+
+
+const fruits12 = ["Apple", "Orange", "Apple", "Mango"];
+let position1 = fruits12.lastIndexOf("Apple") 
+
+
+/*......................................................................................................................................*/
+
+
+const fruits13 = ["Banana", "Orange", "Apple", "Mango"];
+fruits13.includes("Mango"); // is true 
+
+
```

Comment:
Create only one variable declaration if you are using it multiple times.

Reply:
yes, i have make a changes in my Array.js file  by creating  a global variable which is using by every example